### PR TITLE
Convert controller resourcequota files to structured logs

### PR DESCRIPTION
**Pull request by scottilee**
#### What type of PR is this?
/kind cleanup
/sig instrumentation

#### What this PR does / why we need it:
Converts log statements to use structured logging.

#### Which issue(s) this PR fixes:
N/A

#### Special notes for your reviewer:
N/A

#### Does this PR introduce a user-facing change?
```release-note
NONE
```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:
```docs
- [KEP]: https://github.com/kubernetes/enhancements/blob/master/keps/sig-instrumentation/1602-structured-logging/kep.yaml
```

### CODE REVIEW

File: pkg/controller/resourcequota/resource_quota_monitor.go [hunk 1]
```diff
@@ -352,7 +352,7 @@ func (qm *QuotaMonitor) processResourceChanges() bool {
 		utilruntime.HandleError(fmt.Errorf("cannot access obj: %v", err))
 		return true
 	}
-	klog.V(4).Infof("QuotaMonitor process object: %s, namespace %s, name %s, uid %s, event type %v", event.gvr.String(), accessor.GetNamespace(), accessor.GetName(), string(accessor.GetUID()), event.eventType)
+	klog.V(4).InfoS("QuotaMonitor processing object", "obj", event.gvr.String(), "namespace", accessor.GetNamespace(), accessor.GetName(), "UID", string(accessor.GetUID()), "eventType", event.eventType)
```
Comment (yangjunmyfm192085) [inline pkg/controller/resourcequota/resource_quota_monitor.go:355]: I think this is a wrong format?
by the way,you can use klog.KRef.

Comment (scottilee) [reply pkg/controller/resourcequota/resource_quota_monitor.go:355]: Fixed.

**Comment by scottilee** [comment]
@yangjunmyfm192085 I addressed your comment. Let me know if you have any other comments.

**Pull request closed without merging**